app/code/СhaOS/AskQuestion/view/frontend/web/js/askQuestion.js

```diff
@@ -0,0 +1,35 @@
+define([
```

Comment:
This is just a start, but not completed functionality. Please, no not create pull requests till you finish development.